### PR TITLE
Render slices on 3d object

### PR DESCRIPTION
**Pull request by muddv**
Related to #90 
Currently 3d object look like this: 
image
image

And here it is in action:
https://user-images.githubusercontent.com/73133951/227347489-7bea7f0b-9724-4c7c-aefa-5f993c2efe07.mp4

### CODE REVIEW

**Comment by ai** [comment]
I sent it to our designer. For me it looks awesome.

**Comment by ai** [comment]
Looks awesome for both of us. Tomorrow will do a review if we are ready.

File: lib/model.ts [hunk 1]
```diff
@@ -166,6 +218,13 @@ export function initCanvas(
   biggestRgb.listen(value => {
     generateMesh(scene, value)
   })
+  
+  if (!fullControl) {
+    updateSelectors(selectorL, selectorC, selectorH, current.get())
+  }
+  current.listen(value => {
```
Comment (ai) [inline lib/model.ts:225]: Why we need to listen current for cull-screen model (we do not need to draw lines on fullscreen)

Comment (muddv) [reply lib/model.ts:225]: If we are in fullscreen current would not be updating anyway, so it does not influence anything. Shaders are reset to default value, so selectors are not visible in fullscreen

Comment (ai) [reply lib/model.ts:225]: But why we need this line in full-screen? Can we move it to `if (!fullControl)`?

Comment (muddv) [reply lib/model.ts]: Sure, I made the change, also automatically fixed some formatting

**Pull request merged**